test/tools/runner/hooks/configuration.ts

```diff
@@ -204,7 +204,7 @@ const beforeAllPluginImports = () => {
  * @remarks TODO(NODE-4884): once happy eyeballs support is added, we no longer need to set
  * the default dns resolution order for CI
  */
-function installNodeDNSWorkaroundHooks() {
+export function installNodeDNSWorkaroundHooks() {
```

Comment:
Ah this is an oversight from the test filters, I will put up a PR to fix this, we don't need the module.exports at the bottom, it can all be ES syntax.